src/uwtools/scheduler.py

```diff
@@ -0,0 +1,128 @@
+"""
+Job Scheduling
+"""
+
+import logging
+import collections
+from typing import Dict
+
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s %(name)-12s %(levelname)-8s %(message)s",
+    handlers=[logging.StreamHandler()],
+)
+
+
+class JobCard(collections.UserList):
+    """represents a job card to submit to a scheduler"""
+
+    def content(self, line_separator: str = "\n") -> str:
```

Comment:
content includes the "schedulerslurm" information as its first entry, which I believe will cause the job to be rejected on submission. Is the idea that the user will skip that entry when writing out the content to a file?